organization_tags_integration_test.go

```diff
@@ -81,6 +85,27 @@ func TestOrganizationTagsList(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("with query param Query", func(t *testing.T) {
+		tags, err := client.OrganizationTags.List(ctx, orgTest.Name, &OrganizationTagsListOptions{
+			ListOptions: ListOptions{
+				PageNumber: 1,
+				PageSize:   5,
+			},
+			Query: testTagName,
+		})
+		require.NoError(t, err)
+
+		assert.Equal(t, 1, len(tags.Items))
+
+		for _, tag := range tags.Items {
+			assert.Equal(t, tag.Name, testTagName)
+
+			t.Run("ensure org relation is properly decoded", func(t *testing.T) {
+				assert.NotNil(t, tag.Organization)
+			})
+		}
```

Comment:
Use `require.Len(t, tags.Items, 1)` as there is no point in continuing the rest of this test if that is not true. For context, `require` will error and stop the execution of a test, whereas `assert` will not. 

The second item here is the `for` loop is unnecessary as there should only be one item in the list. You also don't need a subtest for ensuring the `Organization` relation is not nil (but I can leave that at your discretion). You can simplify to:
```go
assert.Equal(t, tags.Items[0].Name, testTagName)
assert.NotNil(t, tags.Items[0].Organization)
```

Reply:
@sebasslash,  Thank you for taking the time to review this PR.
I have updated the tests with your feedback, please let me know if this looks good! Cheers!